tier4_system_msgs/srv/OperateMrm.srv

```diff
@@ -0,0 +1,4 @@
+builtin_interfaces/Time stamp
+bool operate
+---
+autoware_adapi_v1_msgs/ResponseStatus response
```

Comment:
Please use [autoware_common_msgs/ResponseStatus](https://github.com/autowarefoundation/autoware_msgs/blob/main/autoware_common_msgs/msg/ResponseStatus.msg) for internal interface services.

Reply:
Fixed to use autoware_common_msgs in the following commit.
https://github.com/tier4/tier4_autoware_msgs/pull/60/commits/97e48f8c8c0c6b65423e871492772400656b1ca4_